tests/test_download.py

```diff
@@ -0,0 +1,80 @@
+from luminoso_api.v5_client import LuminosoClient
+from luminoso_api.v5_download import iterate_docs, download_docs
+
+import pytest
+import requests
+import tempfile
+import json
+
+BASE_URL = 'http://mock-api.localhost/api/v5/'
+RESPONSE = {
+    'result': [
+        {'title': 'Document 1', 'text': 'hello', 'tokens': [{'term_id': 'hello|en'}], 'metadata': [], 'vector': 'AAAA', 'doc_id': 'uuid-0'},
+        {'title': 'Document 2', 'text': 'hello', 'tokens': [{'term_id': 'hello|en'}], 'metadata': [], 'vector': 'AAAA', 'doc_id': 'uuid-1'},
```

Comment:
Our style is to limit lines to 79 characters.

Reply:
Really? Not 99?